src/app.js

```diff
@@ -315,7 +319,7 @@ export const start = async args => {
    * Update current screen with current remote
    */
   const refreshTable = () => {
-    const tableData = state.currentRemote.refs
+    const tableData = ((state.currentRemote && state.currentRemote.refs) || [])
```

Comment:
What did you run into that prompted you to add this?

Reply:
if you give refCount value very low such as `1` , remotes may not have `heads` and 
[here](https://github.com/jwu910/check-it-out/blob/master/src/app.js#L351) the currentRemote will be set to undefined and will result a crash.

Reply:
I think this is an acceptable error and reason to crash. It only crashes if you attempt to select an item that does not exist--but effectively an edge case

The purpose of Check It Out is to change branches and if any user is using `--count=1` there will only be one option to switch to, or no options depending on the first ref returned from the ref getter.

Reply:
are you suggesting we should let it crash @jwu910 ?

Reply:
Yes, the app does not crash on start, it crashes when you are selecting nothing essentially. But you wouldn't use CIO in this manner. The function you linked to is being invoked in a try-catch already.

Reply:
reverted.

Reply:
@jwu910  can you review? its been very long